snapcraft.yaml

```diff
@@ -78,7 +121,6 @@ parts:
     build-snaps:
       - gnome-46-2404-sdk/latest/candidate
 
-
```

Comment:
Cleanups are ok but that doesn't have to do with the commit description, could you split it to a separate commit?

Reply:
Removed.